src/codeu/chat/UserMap.java

```diff
@@ -7,21 +7,30 @@
 import codeu.chat.client.simplegui.NewAccount;
 
 /**
- * @author  Suveena
- * @date    5/24/17
- * @brief   Basic structure that holds all the users. Create this map upon starting GUI. Each time 
- *              we add a new user, check to make sure that they don't already exist. If they do
- *              not, add them to the map.
+ * Basic structure that holds all the users. Creates this map upon starting GUI. Each time we add
+ *         a new user, check to make sure that they don't already exist. If they do not, add them
+ *         to the map.
  */
 public class UserMap {
+    ///////////////////////
+    // PRIVATE VARIABLES //
+    ///////////////////////
+    private static HashMap<String, char[]> map = new HashMap<String, char[]>();
```

Comment:
I don't understand why this is static..... This would be better off as a singleton pattern:

public class UserMap {
  private static UserMap singletonMap = new UserMap();
  private HashMap<> map;
...
  public static UserMap getMap() { return singletonMap; }
}